.github/CODEOWNERS

```diff
@@ -222,7 +222,6 @@ packages/react-components/babel-preset-global-context @microsoft/teams-prg
 packages/react-components/react-table @microsoft/teams-prg
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31007&commitId=b2360076b73216e4560f74614ea7eb145c15529b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Menu Example Divider Horizontal.chromium.png|76| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>